compiler/test/__snapshots__/basic_functionality.52ca8e0e.0.snapshot

```diff
@@ -140,51 +140,25 @@ basic functionality › func_shadow
   (local $6 i32)
   (local $7 i32)
   (return
-   (block $cleanup_locals.18 (result i32)
+   (block $cleanup_locals.16 (result i32)
     (local.set $0
-     (block $compile_block.17 (result i32)
-      (block $compile_store.9
+     (block $compile_block.15 (result i32)
+      (block $compile_store.8
        (global.set $foo_1110
         (tuple.extract 0
          (tuple.make
-          (block $allocate_closure.7 (result i32)
-           (i32.store
-            (local.tee $0
-             (call $malloc_0
-              (global.get $GRAIN$EXPORT$malloc_0)
-              (i32.const 16)
-             )
-            )
-            (i32.const 6)
-           )
-           (i32.store offset=4
-            (local.get $0)
-            (i32.const 1)
-           )
-           (i32.store offset=8
-            (local.get $0)
-            (i32.const -1)
-           )
-           (i32.store offset=12
-            (local.get $0)
-            (i32.const 0)
-           )
-           (local.get $0)
-          )
+          (i32.const 0)
```

Comment:
Is this now one of those tests where we are optimizing the whole thing away?

Reply:
The whole closure gets optimized away here (which shows that the feature is working!), but not the rest of the test.